types/nodemailer-stub/index.d.ts

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
nodemailer-stub doesn't have a typescript module types avaiable in npm, hence create one manually